.github/workflows/sub-ci-integration-tests-gcp.yml

```
@@ -165,7 +162,7 @@ jobs:
       app_name: zebrad
       test_id: sync-update-mainnet
       test_description: Test syncing to tip with a Zebra tip state
-      test_variables: "NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=sync-update-mainnet,ZEBRA_CACHE_DIR=/home/zebra/.cache/zebra"
+      test_variables: "ZEBRA_NETWORK__NETWORK=${{ inputs.network || vars.ZCASH_NETWORK }},NEXTEST_PROFILE=sync-update-mainnet"
```

We could probably remove the `ZEBRA_NETWORK__NETWORK` definition here altogether, but, if we're keeping it, the `needs.sync-full-mainnet.result == 'success'` condition above implies this will always be Mainnet.

```suggestion
      test_variables: "ZEBRA_NETWORK__NETWORK=Mainnet,NEXTEST_PROFILE=sync-update-mainnet"
```

Update: Not a blocker.